cfme/tests/containers/test_basic_metrics.py

```diff
@@ -8,8 +8,11 @@
     pytest.mark.tier(1),
     pytest.mark.provider([ContainersProvider], scope='function')]
 
+# TODO This test needs to be reevaluated. This is not testing anyting in CFME.
+
 
 @pytest.mark.polarion('CMP-10205')
+@pytest.mark.uncollectif(lambda provider: provider.metrics_type == 'Prometheus')
```

Comment:
We can consider parameterization on this test so it would support any provider type, but on failure, it would raise an exception on specific endpoint type.